tests/integration/targets/lightsail_static_ip/tasks/main.yml

```diff
@@ -0,0 +1,68 @@
+---
+
+- module_defaults:
+    group/aws:
+      aws_access_key: '{{ aws_access_key | default(omit) }}'
+      aws_secret_key: '{{ aws_secret_key | default(omit) }}'
+      security_token: '{{ security_token | default(omit) }}'
+      region: '{{ aws_region | default(omit) }}'
+
+  block:
+
+    # ==== Tests ===================================================
+
+    - name: Create a new static IP
+      lightsail_static_ip:
+        name: "{{ static_ip_name }}"
+        region: "{{ region }}"
+      register: result
+
+    - assert:
+        that:
+          - result.changed == True
+          - "'static_ip' in result and result.static_ip.name == static_ip_name"
+          - "'static_ip' in result and result.static_ip.resourceType == 'StaticIP'"
+          - "result.instance.state.name == 'running'"
```

Comment:
`allocate_static_ip_address` does not return a state key.  It does return a `status` but the module does not directly return this response. The keys being returned by `get_static_ip` that can be asserted look like this:
```
    "static_ip": {
        "arn": "arn:aws:lightsail:us-east-1:966509639900:StaticIp/9658cdc0-fef3-4fdd-83f3-2a438a8608a4",
        "created_at": "2021-01-14T21:38:48.416000+00:00",
        "ip_address": "54.236.109.178",
        "is_attached": false,
        "location": {
            "availability_zone": "all",
            "region_name": "us-east-1"
        },
        "name": "ansible-test-lab-36684871_static_ip",
        "resource_type": "StaticIp",
        "support_code": "011036789062/54.236.109.178"
    }
```